app/assets/stylesheets/global.css

```diff
@@ -36,7 +36,6 @@ button, .button {
 
 /* Display Helpers */
 .no-margin { margin: 0; }
-.noheight { font-size: 0; }
```

Comment:
… oops. I didn't mean to remove this; I accidentally added a thing and removed the wrong thing.